src/components/Profile/ProfileInfoCard/index.js

```diff
@@ -279,11 +289,17 @@ const ProfileInfoCard = () => {
                   </Box>
                 </span>
               </p>
-              <Box mr={12}>{checkAvailable(profileData.description) && <p className="text-justified">{profileData.description}</p>}</Box>
+              <Box mr={12}>
+                {checkAvailable(profileData.description) && <p className="text-justified">{profileData.description}</p>}
+              </Box>
 
               {checkAvailable(profileData.link_facebook) && (
                 <p>
-                  <a href={"https://www.facebook.com/" + profileData.link_facebook} target="_blank" rel="noopener noreferrer">
+                  <a
+                    href={"https://www.facebook.com/" + profileData.link_facebook}
```

Comment:
Shall we use const variables for these URL prefixes. This will not affect the functionality, but will increase the readability.

Reply:
Thank you for mentioning that. I committed with const for URIs.